Doc/whatsnew/3.15.rst

```diff
@@ -369,6 +355,19 @@ tarfile
   (Contributed by Matt Prodani and Petr Viktorin in :gh:`112887`
   and :cve:`2025-4435`.)
 
+tkinter
+-------
+
+* The ``tkinter.Text.search`` method now supports two additional
+  arguments: ``-nolinestop`` which allows the search to continue across line boundaries;
+  and ``-strictlimits`` which restricts the search to within the specified range.
```

Comment:
If we talk about a Tkinter method instead of a Tcl/Tk command, argument names do not have initial hyphens.

One line also exceeds the recommended limit for rst files.